### PR TITLE
Edit Azure Functions integration article

### PR DESCRIPTION
**Pull request by rkoron007**
I didn't change any of the code— just tweaked and rephrased the written content.

### CODE REVIEW

File: docs/source/deployment/azure-functions.mdx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+If you navigate to [http://localhost:7071/api/graphql?name=Apollo](http://localhost:7071/api/graphql?name=Apollo) in your browser and will now see the text: 'Hello Apollo.'
+
+<ExpansionPanel title="Run into a problem?">
+
+If you see an error message mentioning "Incompatible Node.js version", you should double-check that the version of `node` on your `PATH` is a version [supported by Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=in-process%2Cv4&pivots=programming-language-javascript). If your node version is not supported, you can use a tool like [nvm](https://github.com/nvm-sh/nvm) to use a different node version.
+
+</ExpansionPanel>
+
+If you'd like to change the URL for this function (e.g., to something like `http://{my-url}/graphql`), you can change the URL prefix in the `host.json` file:
```
Comment (glasser) [inline docs/source/deployment/azure-functions.mdx:61]: I think that no longer mentioning that the point of this suggestion is "remove `/api`" makes this part more confusing.

I also think that the curlies around `{my-url}` are confusing (not new with this change); using a domain like my-app.example.com might be better.

**Review by glasser — COMMENTED** [review]
I'm not sure I was able to notice all of the changes but things look good on the whole; that said I don't know much about AF anyway.

File: docs/source/deployment/azure-functions.mdx [hunk 2]
```diff
@@ -0,0 +1,219 @@
+---
+title: Deploying with Azure Functions
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:2]: I might suggest naming this "Deploying Apollo Server to Azure Functions" and omitting the description (I bet we use this current pattern for all these deployment articles, but that feels incrementally better to me

File: docs/source/deployment/azure-functions.mdx [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+We created this tutorial using a Linux environment. Some commands might differ if you use a Windows environment; [please refer to the Azure documentation for more details](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli).
+
+Note that `apollo-server-azure-functions` does not enable you to add arbitrary middleware to your web server (though you can manually wrap the handler returned by `createHandler` in your own handler).
+
+## Prerequisites
+
+You must do following before proceeding:
+
+- Setup an [Azure](https://azure.com) account
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:21]: "Setup" -> "Set up"

File: docs/source/deployment/azure-functions.mdx [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+- Install the [Azure Functions Core Tools CLI version 3.x](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#v2)
+- Install the [Azure CLI 2.x](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+
+## Setting up the project
+
+### Creating a new Function
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:27]: lowercase function I reckon

File: docs/source/deployment/azure-functions.mdx [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+[2021-06-30T19:22:21.077Z] Worker process started and initialized.
+
+Functions:
+
+	graphql: [GET,POST] http://localhost:7071/api/graphql
+
+For detailed output, run func with --verbose flag.
+```
+
+If you navigate to [http://localhost:7071/api/graphql?name=Apollo](http://localhost:7071/api/graphql?name=Apollo) in your browser and will now see the text: 'Hello Apollo.'
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:53]: "Navigate to [url] in your browser. It displays the text `Hello Apollo`.

File: docs/source/deployment/azure-functions.mdx [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+<ExpansionPanel title="Run into a problem?">
+
+If you see an error message mentioning "Incompatible Node.js version", you should double-check that the version of `node` on your `PATH` is a version [supported by Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=in-process%2Cv4&pivots=programming-language-javascript). If your node version is not supported, you can use a tool like [nvm](https://github.com/nvm-sh/nvm) to use a different node version.
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:57]: "you should double-check" -> "confirm"

`node` -> Node.js (in all spots here)

"to use a different node version" -> "to use a different version"

File: docs/source/deployment/azure-functions.mdx [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+
+Before deploying, we first need to meet the requirements to set up a new [Azure Function](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview).
+
+We'll start by creating a new [Azure resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal) for this project. The below command will create a resource group with the `--name` and `--location` you specify. Make sure you replace the `--location` with [your Azure region](https://azure.microsoft.com/en-us/global-infrastructure/geographies/).
+
+```shell
+az group create --name apollo-examples --location eastus
+```
+
+After creating a resource group, you will need to create a [storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) to store your code on Azure. Make sure to choose a **unique** name for your project's storage account.
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:161]: "will need" -> "need"

File: docs/source/deployment/azure-functions.mdx [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+
+```shell
+az storage account create \
+    --name apolloexampleYOURNAME \
+    --location eastus \
+    --resource-group apollo-examples \
+    --sku Standard_LRS
+```
+
+The last resource you'll need to create is a [function app](https://docs.microsoft.com/en-us/cli/azure/functionapp?view=azure-cli-latest#az-functionapp-create). The below command will create a new function app using the specified resource group, location, and storage account.
```
Comment (StephenBarlow) [inline docs/source/deployment/azure-functions.mdx:171]: "will create" -> "creates"

**Review by StephenBarlow — APPROVED** [review]
Approved with small suggestions!

**Pull request merged**